packages/core/src/components/cv-search/cv-search.vue

```diff
@@ -114,6 +115,17 @@ export default {
     },
   },
   computed: {
+    wrapperAttrs() {
+      const attrs = {};
+      if (this.ariaLabel) {
+        attrs['aria-label'] = this.ariaLabel;
+      } else if (this.label) {
+        attrs['aria-labelledby'] = this.uid;
+      } else {
+        attrs['aria-label'] = "Search box";
```

Comment:
Hard coded string

Reply:
That provides a default "arial-label" for the search box if the user did not pass in arialLable. I am ok to remove that.  Please let me know if this is your intention (to not have adefault arial-labels value).  Thanks